main.cpp

```diff
@@ -52,85 +52,85 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: recorder
+//  action 1: set sample rate
+recorder.setSampleRate();
+//  action 2: set bit rate
+recorder.setBitRate();
+//  action 3: select channel
+recorder.selectChannel();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: human
+//  action 1: eat
+human.eat();
+//  action 2: sleep
+human.sleep();
+//  action 3: write
+human.write();
+//  3) 
+//  Noun: reverb
+//  action 1: choose ir data
+reverb.chooseIR();
+//  action 2: set wet amount
+reverb.setWetAmount();
+//  action 3: set distance
+reverb.setDistance();
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: audio channel 1
+//  action 1: import file
+channel1.importFile();
+//  action 2: solo
+channel1.solo();
+//  action 3: record sound
+channel1.recSound();
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: washing machine
+//  action 1: set quick wash
+washer.setQuickWash();
+//  action 2: set cottons
+washer.setCottons();
+//  action 3: stop washing
+washer.stopWashing();
+//  6) 
+//  Noun: browser
+//  action 1: add newaddNtab
+browser.addNewTab();
+//  action 2: refresh page
+browser.refresh();
+//  action 3: close window
+browser.closeWindow();
+//  7) 
+//  Noun: bluetooth device
+//  action 1: sand a file
+bt.sendFile();
+//  action 2: receive a file
+bt.receiveFile();
+//  action 3: pair device
+bt.pairDevice();
 //  8)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: limiter
+//  asetAtion 1: attack time
+limiter.setAttackTime(setH;
+//  action 2: hold
+limiter.setHold();
+//  action 3: limit
+limiter.setLimit();
 //  9)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: midi keyboard
+//  action 1: get note number
+midi.getNoteNumber();
+//  action 2: get pitch wheel value
+midi.getPitchWheelValue();
+//  action 3: control number
+midi.getControllerNumber();

```

Comment:
As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.